driver-core/src/main/com/mongodb/client/model/expressions/DateExpression.java

```diff
@@ -19,22 +19,135 @@
 import java.util.function.Function;
 
 /**
- * Expresses a date value.
+ * A UTC date-time {@linkplain Expression value} in the context
+ * of the MongoDB Query Language (MQL). Tracks the number of
+ * milliseconds since the Unix epoch, and does not track the timezone.
  */
 public interface DateExpression extends Expression {
+
+    /**
+     * The year of {@code this} date as determined by the provided
+     * {@code timezone}.
+     *
+     * @param timezone the UTC Offset or Olson Timezone Identifier.
```

Comment:
Let's either give examples of both the offset and the identifier values, or link to the relevant part of the MongoDB documentation. I am fine with either duplicating this information everywhere, or specifying it in the doc comment for `DateExpression`.

Reply:
Added a link

Reply:
The added link is about the format specifier, not the offset.